tests/frontend/runner.js

```diff
@@ -1,166 +1,4 @@
 $(function(){
```

Comment:
Less code!  Awesome :)

Reply:
Yes, the runner now is running Mocha pretty straightforward, but the responsibility is now with Selenium on `remote_runner` to parse the results :p